plugins/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -153,6 +153,10 @@
         type: bool
         default: yes
         version_added: 1.3.0
+    priority:
+        description:
+          - Integer from 1 (high) to 5 (low) indicating alert severity.
+        type: str
```

Comment:
```suggestion
        type: int
```
If it is an integer, this should have `type=int`. Also if there are just five possibilities, it's better to add `choices`:
```suggestion
        type: str
        choices: [1, 2, 3, 4, 5]
```

Reply:
Let's not declare choices. Better to follow your suggestion to keep as int and then the DD API or the DD documentation take care of choices.